dockerfiles/Dockerfile.tensorrt

```diff
@@ -12,7 +12,7 @@ ARG ONNXRUNTIME_REPO=https://github.com/Microsoft/onnxruntime
 ARG ONNXRUNTIME_BRANCH=main
 # Adjust as needed 
 # Check your CUDA arch: https://developer.nvidia.com/cuda-gpus
-ARG CMAKE_CUDA_ARCHITECTURES=75;80;90
+ARG CMAKE_CUDA_ARCHITECTURES=75-real;80-real;86-real;89-real;90-real;100-real;120
```

Comment:
May need to update the TRT_CONTAINER_VERSION in line 8 to the latest one.

Reply:
reverted